app-emulation/atari800_libretro/atari800_libretro-3.1.0_20200429.recipe

```diff
@@ -27,6 +32,7 @@ REQUIRES="
 	"
 
 BUILD_REQUIRES="
+	git$secondaryArchSuffix
```

Comment:
Is this really needed?

Reply:
Maybe not. I added it for tests but if does build without it. I'll remove it.

Reply:
Removed in [#067d672](https://github.com/haikuports/haikuports/pull/4966/commits/067d672708ed5b492f8957d9229fa64fe76fa383)